### ISSUE TITLE
Error C1047 when building Release x64

### ISSUE THREAD

**Issue by mlmeira** [xlw/xlw#11]
Recently installed MSVS 2019, simple "test" xll creation , works fine for debug and win32 releases, but won't link the release/x64 build: 
fatal error C1047: The object or library file 'C:\Users\...\packages\xlw.20.8.16-DEVd0cc\build\..\native\lib\x64\\xlw-vc142-mt-20_08_16.lib' was created by a different version of the compiler than other objects like 'x64\Release\Black.obj'; rebuild all objects and libraries with the same compiler

**Comment by keithalewis** [comment]
This sounds like a NuGet issue. The xlw package providers need to build a version using MSVC 2019.
If you download the source code you can build that for yourself and add a reference in Solution Explorer.

**Comment by MarcoPolo314** [comment]
Hey folks, I appear to be having the exact same issue.  The solution builds fine for every combination *except* Release + x64, which is the one I need, of course.  Have you guys been able to find a work around yet?

**Comment by MarcoPolo314** [comment]
Hey folks, does anyone know if there is a way to find out which version of the compiler "xlw-vc142-mt-20_08_16.lib" was created in, and if there's a way to set Visual Studio to match it so that a Release x64 can build without error?

@mlmeira did you ever manage to solve this problem?

Alternatively @keithalewis , is it very difficult to build the the solution from source code as you suggest?  I'm a n00b and don't know how to do that.  

If anybody can help, thanks in advance.

**Comment by keithalewis** [comment]
@MarcoPolo314 Let me know if you run into problems following the directions in the README.

**Comment by MarcoPolo314** [comment]
Hey @keithalewis , which README file are you looking at?  I don't see any directions for rebuilding the source code in any README file I can see.

**Comment by keithalewis** [comment]
@MarcoPolo314  See https://github.com/xlladdins/xll/blob/master/README.md
Open an issue there if you run into problems.

**Comment by MarcoPolo314** [comment]
Is xll a completely different project from xlw?

**Comment by MarcoPolo314** [comment]
Is xll a completely different project from xlw?  @keithalewis

**Comment by mlmeira** [comment]
@MarcoPolo314 yes, I follows @keithalewis recommendation and built the whole package from source code.

**Comment by MarcoPolo314** [comment]
@mlmeira would you mind terribly posting instructions on how to do that?  I'm not sure how.

**Comment by keithalewis** [comment]
@MarcoPolo314 Yes. My approach is to provide a minimal C++ wrapper around the Excel SDK. It is not as full-featured as xlw.

**Comment by isaiahpantelis** [comment]
Could someone possibly clarify the steps for setting up the properties of a project, after having built xlw from source? I cloned xlw, built it, and then added to the `Include Directories` and `Library Directories` of my project the directories that contain the header files and .lib files of xlw, respectively; however, I'm getting (eight) unresolved external symbol errors. Thanks in advance.

**Comment by isaiahpantelis** [comment]
Could someone possibly clarify the steps for setting up the properties of a project, after having built xlw from source? I cloned xlw, built it, and then added to the Include Directories and Library Directories of my project the directories that contain the header files and .lib files of xlw, respectively; however, I'm getting (eight) unresolved external symbol errors. Thanks in advance.

**Comment by GaryVel** [comment]
Have a solution to the above release / x64 problem: 

In Properties | C/C++ | Optimization, set "Whole Program Optimization" to no.

More detail and explanations here:
https://stackoverflow.com/questions/837073/error-c1047-object-file-created-with-an-older-compiler-than-other-objects/17429775

**Comment by GaryVel** [comment]
> Hey folks, does anyone know if there is a way to find out which version of the compiler "xlw-vc142-mt-20_08_16.lib" was created in, and if there's a way to set Visual Studio to match it so that a Release x64 can build without error?
> 
> @mlmeira did you ever manage to solve this problem?
> 
> Alternatively @keithalewis , is it very difficult to build the the solution from source code as you suggest? I'm a n00b and don't know how to do that.
> 
> If anybody can help, thanks in advance.

Sorry, meant to tag your question in my reply above. Clearly need extra coffee this morning.
